scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/Scheduler.java

```diff
@@ -449,7 +449,7 @@ public void setConfigurationFile(String filename) {
   public synchronized void start() {
     if (scanDirectoriesFuture == null) {
       scanDirectoriesFuture =
-          scheduler.scheduleAtFixedRate(
+          scheduler.scheduleWithFixedDelay(
```

Comment:
Does this fix something? Doesn't it mean that Run Scanner would take even longer to get through all the runs after a restart?

Reply:
I'm hoping it will. There's a pattern in the Prometheus graphs that makes me think it's lapping itself, which isn't helpful. I'd like to avoid that. It shouldn't take any longer after start up. This will still process all the directories after the first scan.